applications/incompressible_navier_stokes/orr_sommerfeld/include/perturbation_energy.h

```diff
@@ -134,7 +133,9 @@ class PerturbationEnergyCalculator
 
 private:
   void
-  do_evaluate(VectorType const & velocity, double const time, unsigned int const time_step_number)
+  do_evaluate(VectorType const &     velocity,
+              double const           time,
+              types::time_step const time_step_number)
   {
     if((time_step_number - 1) % data.calculate_every_time_steps == 0)
```

Comment:
I think we would also have to adjust this code here.

Reply:
done